gnome-abrt.spec.in

```diff
@@ -77,36 +76,27 @@ desktop-file-install \
 make check
 
 
-%post
-# update icon cache
-touch --no-create %{_datadir}/icons/hicolor &>/dev/null || :
-
-
-%postun
-if [ $1 -eq 0 ] ; then
-    touch --no-create %{_datadir}/icons/hicolor &>/dev/null
-    gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
-fi
-
-
-%posttrans
-gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
-
```

Comment:
This change of post scriplets are relevant for fedoras > 27. You should add conditional

%if 0%{?fedora} > 27
# post scriplets are not needed
%else
...
%endif

Reply:
@mhabrnal no, this is wrong. You should do `%if 0%{?rhel} && 0%{?rhel} <= 7`..

But I would remove it entirely.

Reply:
@ignatenkobrain  so, these post scriplets are no longer needed in Fedora 26 and Fedora 27?

Reply:
just FYI, F26 is dead.. Really.

But yes, they are not needed since F26/